examples/13_DTQP/IEA-15-floating.yaml

```diff
@@ -0,0 +1,1206 @@
+name: IEA 15MW Offshore Reference Turbine, with taped chord tip design

```

Comment:
Does this file need to be here as opposed to using the examples/06 version? Specifically, is this yaml different than that one?

Reply:
If it's the same, I'd suggest removing this one and referencing a singular geometry yaml.

Reply:
It should be the same as the geometry in example 12, so I agree, referencing a singular geometry yaml is best. Pushed!